src/library/scala/annotation/apiStatus.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Scala (https://www.scala-lang.org)
+ *
+ * Copyright EPFL and Lightbend, Inc.
+ *
+ * Licensed under Apache License 2.0
+ * (http://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package scala.annotation
+
+import scala.annotation.meta._
+
+/**
+ * An annotation to denote the API status.
+ *
+ * @param  message the advisory to print during compilation
+ * @param  category a string identifying the categorization of the restriction
+ * @param  since a string identifying the first version in which the status is applied
+ * @param  defaultAction the default severity of the restriction when the annotee is referenced
+ * @since  2.13.2
+ * @see    [[scala.annotation.apiStatus.Action]]
+ * @see    [[scala.annotation.apiStatus.Category]]
+ */
+@getter @setter @beanGetter @beanSetter @companionClass @companionMethod
+class apiStatus(
+  message: String,
+  category: String,
+  since: String = "",
+  defaultAction: String = apiStatus.Action.Warning) extends scala.annotation.StaticAnnotation
+
+object apiStatus {
+  object Action {
+    final val Error = "error"
+    final val Warning = "warning"
+    final val WarningSummary = "warning-summary"
+    final val WarningVerbose = "warning-verbose"
```

Comment:
Probably we should not expose the "verbose" options here?

Reply:
👍

Reply:
Fixed in https://github.com/scala/scala/pull/8820/commits/9f6413c4e9fcef749bc38040e250706df9b82d21. Also added Scaladoc in there.